src/Loggers/MassTransit.SerilogIntegration/SerilogLog.cs

```diff
@@ -0,0 +1,269 @@
+﻿// Copyright 2007-2016 Chris Patterson, Dru Sellers, Travis Smith, et. al.
+//  
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the 
+// License at 
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0 
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR 
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the 
+// specific language governing permissions and limitations under the License.
+namespace MassTransit.SeriLogIntegration
+{
+    using System;
+    using System.Collections.Generic;
+    using Logging;
+    using Serilog.Events;
+
+
+    public class SerilogLog : ILog
+    {
+        public const string ObjectLogTemplate = "{@obj}";

```

Comment:
Serilog uses uppercase property names by convention, so `Obj` or `Object` might fit a little closer to other events.

Unless all of the call sites are careful about what object is passed, serializing `@` might be a problematic default due to the potential payload size if the object's not a tidy DTO. (@jaben probably has experience with the variety of arguments that come through the pipeline...)